### ISSUE TITLE
In-built password generator

### ISSUE THREAD

**Issue by pedrosimao** [subdavis/Tusk#80]
## This issue is a 

- [ ] bug
- [x] feature
- [ ] question

### Please describe how you think it should change.

It would be nice to have an in-built password generator, where we could choose the number of characters and the kind of character of our random passwords. Like this we could be able to generate strong passwords to our new accounts or updated accounts. It is super nice to have all inside one single extension. Like a password manager swiss knife.

**Comment by zmilonas** [comment]
A lot of people might not know this but in case of Chrome this feature is already built into the browser.

https://support.google.com/chrome/answer/7570435?hl=en

For firefox the most I can recommend is LastPass' password generation website: https://www.lastpass.com/password-generator

**Comment by zmilonas** [comment]
But that has nothing to do with this issue

**Comment by subdavis** [comment]
I think you're both right.  

This issue is exclusively to track a password generator, but a generator is a little pointless without an ability to persist the new password.

I'm going to treat this issue as "password generator given some way to save it".

**Comment by pedrosimao** [comment]
@zmilonas @subdavis You are right. I opened the issue about password generator because there was already another open issue about persisting passwords. I would be glad to help you implement the feature whenever I will have some free time.
